.github/workflows/publish.yml

```diff
@@ -147,3 +147,62 @@ jobs:
            GITHUB_TOKEN: ${{ github.token }}
         run: |
             gh release upload ${{github.event.release.tag_name}} ./extension/playwright-mcp-extension-${{ steps.get-version.outputs.version }}.zip
+
+  publish-release-mcp-registry:
+    if: github.event_name == 'release'
+    needs: publish-release-npm
+    runs-on: ubuntu-latest
+    environment: allow-mcp-registry-publishing
+    permissions:
+      contents: read
+      id-token: write # Needed for GitHub OIDC authentication
+    steps:
+      - uses: actions/checkout@v5
+      - name: Clone MCP Registry and build publisher tool
+        shell: pwsh
+        run: |
+          cd ${{ runner.temp }}
```

Comment:
Can we just checkout and build in the default workspace?

Reply:
I thought it was best to keep the tool compilation out of the main workspace since it's unrelated to the MCP package content. I feel like there is a small risk of the npm packaging at some point including a `registry` subdirectory if the build changes. But if you feel strongly, we can do that.